### ISSUE TITLE
Top Hero Bar is broken in 12 vs 12

### ISSUE THREAD

**Issue by darklordabc** [darklordabc/10v10_plus#4]
![image](https://user-images.githubusercontent.com/16277198/49152463-d1360100-f30a-11e8-9709-11a3690ce7ef.png)

**Comment by toomasluts1** [comment]
Yeah, noticed too. To be more specific... what I think is going on is that people who have permanently left the game don't get displayed on hero bar at all, their icon just dissapears rather than turning red.

**Comment by darklordabc** [comment]
This should be fixed now

**Comment by toomasluts1** [comment]
Apparently not. I played just now and a leaver's icon still dissapeared.

**Comment by darklordabc** [comment]
Yeah I don't know, this could actually be a good thing. Seeing the disconnected players is discouraging, and gives the impression to players that they are going to lose, I don't really see the benefit in players seeing the disconnected players, but I do see the discouragement that seeing them brings.

**Comment by ZackNovazich** [comment]
cuz this is 12v12 game, 7.20d version is quick-push version, any team which isn't 12 players is in disadvantage that u barely win the game.btw i'm in china we could restart a game quickly!!!!

**Comment by ZackNovazich** [comment]
switch to the losing team ? u could have a try,but how do u know the team is losing?  by the kills , the gold,or the xp?

**Comment by darklordabc** [comment]
It would simply by hero count. Players from the team that has 2 more players than the other team, will be able to switch to the enemy team. Its possible that the team with less players is actually winning, but that's pretty unlikely, and wont be considered in the calculation.

**Comment by ZackNovazich** [comment]
oh i see, btw could u add update log in chinese ?  not any chinese could understand english，like silcencer don't steal int in chinese is 沉默术士不偷取智力

**Comment by darklordabc** [comment]
The changelog has been update dot have google translated simplifed chinese https://steamcommunity.com/sharedfiles/filedetails/changelog/1576297063

**Comment by ZackNovazich** [comment]
i saw it , maybe u dont know  we could visit steamcommunity only through vpn, the GFW(Great Fire Wall) blocked the  steamcommunity,this is so fucked up in china. i'll appreciate that show the changelog in chinese  in the game.
